packages/twenty-front/src/modules/object-record/record-inline-cell/components/RecordInlineCellContainer.tsx

```diff
@@ -62,7 +62,7 @@ const StyledClickableContainer = styled.div`
 const StyledInlineCellBaseContainer = styled.div`
   align-items: center;
   box-sizing: border-box;
-
+  width: 100%;
```

Comment:
Thanks for the PR! This broke the centering for other elements (look at the Title on the record of a show page for example)